it/core/src/main/java/com/adobe/cq/forms/core/components/it/servlets/FileAttachmentServlet.java

```diff
@@ -45,7 +45,7 @@
 @Component(service=Servlet.class,
         property={
                 "sling.servlet.methods=" + HttpConstants.METHOD_GET,
-                "sling.servlet.resourceTypes="+ "fd/af/components/guideContainer",
+                "sling.servlet.resourceTypes="+ "core/fd/components/form/container/v2/container",
                 "sling.servlet.selectors=" + FileAttachmentServlet.SELECTOR
         })
 @ServiceDescription("Adaptive Form File Attachment Retrieve Servlet")
```

Comment:
can't recall why its part of IT

Reply:
@nit23uec I had the same question when I was trying to use `FormConstants` to import this path. Tbh I'm not sure why the Prefill and Submit services are a part of IT either.
@ravise5 Could you pitch in?

Reply:
This is just a sample to test file attachment preview from server, its required